packages/fluentui/react-northstar/src/components/Form/FormFieldCustom.tsx

```diff
@@ -0,0 +1,175 @@
+    const labelId = React.useRef<string>();
+    labelId.current = getOrGenerateIdFromShorthand('form-label-', id, labelId.current);
+
+    const getA11yProps = useAccessibility<FormFieldBehaviorProps>(props.accessibility, {
+      debugName: composeOptions.displayName,
+      mapPropsToBehavior: () => ({
+        hasErrorMessage: !!errorMessage,
+        messageId: messageId.current,
+        labelId: labelId.current,
+      }),
+      rtl: context.rtl,
+    });
+
+    const { classes } = useStyles<FormFieldCustomStylesProps>(FormFieldCustom.displayName, {
+      className: composeOptions.className,
+      composeOptions,
+      mapPropsToStyles: () => ({
+        inline,
+        hasErrorMessage: !!errorMessage,
+      }),
+      mapPropsToInlineStyles: () => ({
+        className,
+        design,
+        styles,
+        variables,
+      }),
+      rtl: context.rtl,
+      unstable_props: props,
+    });
+
+    const labelElement = createShorthand(composeOptions.slots.label, label, {
+      defaultProps: () =>
+        getA11yProps('label', {
+          htmlFor: id,
+          id: labelId.current,
+          inline,
+          ...slotProps.label,
+        }),
+    });
+
+    const messageElement = createShorthand(composeOptions.slots.message, errorMessage || message, {
+      defaultProps: () =>
+        getA11yProps('message', {
+          id: messageId.current,
+          ...slotProps.message,
+        }),
+    });
+
+    const content = (
+      <>
+        {labelElement}
+        {(slotProps.root as any).children}
+        {messageElement}
+      </>
+    );
+
+    const element = (
+      <ElementType
+        {...getA11yProps('root', {
+          className: classes.root,
+          ref,
+          ...slotProps.root,
+        })}
+      >
+        {childrenExist(children) ? children : content}
+      </ElementType>
+    );
+    setEnd();
+    return element;
+  },
+  {
+    className: formFieldClassName,
+    displayName: 'FormFieldCustom',
+    overrideStyles: true,
+    slots: {
+      label: FormLabel,
+      message: FormMessage,
+      control: Box,
+    },
+    handledProps: [
+      'accessibility',
+      'inline',
+      'as',
+      'children',
+      'className',
+      'design',
+      'design',
+      'errorMessage',
+      'message',
+      'variables',
+      'styles',
+    ],
+    shorthandConfig: {},
+  },
+);
+
+FormFieldCustom.propTypes = {
+  ...commonPropTypes.createCommon({
+    content: false,
+  }),
+  // control: customPropTypes.shorthandAllowingChildren,
+  id: PropTypes.string,
+  inline: PropTypes.bool,
+  message: customPropTypes.itemShorthand,
+  errorMessage: customPropTypes.shorthandAllowingChildren,
+};
+
+FormFieldCustom.defaultProps = {
+  accessibility: formFieldBehavior,
+};
+
+/**
+ * A FormFieldCustom represents a Form element containing a label and an input.
+ */
+export default withSafeTypeForAs<typeof FormFieldCustom, FormFieldCustomProps>(FormFieldCustom);
```

Comment:
```suggestion
export default FormFieldCustom;
```

We don't need this for `FormFieldCustom`